### ISSUE TITLE
GET request over file protocol glitch

### ISSUE THREAD

**Issue by pal1000** [hackademix/noscript#147]
Create a HTML document and open it over file protocol with a random GET request payload, per testcase itself:
````
<html>
<head>
<title>NoScript - GET request over file protocol glitch</title>
</head>
<body>
<script>
document.write('Open this page with a random GET request and this active content won&apos;t run unless you allow whole file protocol');
</script>
</body>
</html>
````
Reproduced with Firefox on Windows.

**Comment by jtotht** [comment]
@pal1000 How do you expect it to behave? NoScript block/allow is based on domains; there is no concept of domains in the `file` protocol. On Windows drives could be taken into consideration (i.e. one could allow `file:///c/*` while keeping `file:///d/*` disabled), which is kind of similar to domains, but I think it’s even more counter-intuitive than the current situation, and not platform-independent (there are no drives on Unix-like OSs).

**Comment by pal1000** [comment]
I think NoScript should look for `?` in file URLs then ignore it and everything after it when checking permissions. So absolute path without GET request payload should be the domain name equivalent for file protocol. At least this is how NoScript behaves when there is no GET request involved. I want NoScript to behave the same with GET request.

**Comment by pal1000** [comment]
This is no longer an issue. Tested 11.0.44. I think it was fixed in 11.0.42rc3.
